config/packages/security.yaml

```diff
@@ -33,14 +33,15 @@ security:
                     - OAT\SimpleRoster\Security\Authenticator\ApiKeyAuthenticator
             stateless: true
 
-        lti1p3:
-            pattern: '^%app.route_prefix%/v1/lti1p3/outcome'
+        platform_service_basic_outcome_area:
+            pattern: '%app.route_prefix%/v1/lti1p3/outcome'
             stateless: true
             provider: app_user_provider
-            lti1p3_service:
-                scopes:
-                    - 'https://purl.imsglobal.org/spec/lti-bo/scope/basicoutcome'
-
+            lti1p3_service: {
+                scopes: [
+                        'https://purl.imsglobal.org/spec/lti-bo/scope/basicoutcome'
+                ]
```

Comment:
All other arrays in the file are using the `- Item` notation, why change this one?

Reply:
It doesn't work with the other format for some reason.